src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -46,7 +62,71 @@ public EiffelPluginConfiguration(final String pluginName, final NameKey project,
             throw new ExceptionInInitializerError(String.format("Can't read %s plugin configuration for project %s: %s",
                     pluginName, project.toString(), e.getMessage()));
         }
+        // Read plugin configuration
+        this.enabled = pluginConfig.getBoolean(ENABLED, false);
+        this.filter = pluginConfig.getString(FILTER);
+        this.remremPublishURL = pluginConfig.getString(REMREM_PUBLISH_URL);
+        this.remremUsername = pluginConfig.getString(REMREM_USERNAME);
+        this.remremPassword = pluginConfig.getString(REMREM_PASSWORD);
+        //flow context is optional
+        this.flowContext = pluginConfig.getString(FLOW_CONTEXT);
 
+        // No point to check other config parameters if plugin is disabled
+        boolean isValidated = validatePluginConfig(pluginName, project);
```

Comment:
I think "isValidated" is the wrong word. I would probably say, isValid or isConfigValid to make it more clear. :)

Reply:
If my above suggestion is followed, the boolean can be skipped. The throws will handle any exceptions from validate.

Reply:
I have removed most of the validations, and just kept the Publish URL as the mandatory one.